README.md

```diff
@@ -31,6 +31,7 @@ None
 | `jenkins_master_plugins` | plugins to install | `[]` |
 | `jenkins_master_ssh_passphrase` | passphrase of ssh key | `""` |
 | `jenkins_master_ssh_private_key` | ssh private key | `""` |
+| `jenkins_master_nodes` | slave nodes | `[]` |
```

Comment:
would you describe what the variable expects? especially when one expects structured values like this. is `name` a FQDN? used for what? what does `remotefs` do? does it work if `host` is not an IP address? that kind of questions should be answered in `README`